Permanent/ViewControllers/TagManagementViewController.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  TagManagementViewController.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 11.10.2022.
+//
+
+import UIKit
+
+class TagManagementViewController: BaseViewController<ManageTagsViewModel> {
+    
+    @IBOutlet weak var archiveTitleNameLabel: UILabel!
+    @IBOutlet weak var archiveTitleTagsNbrLabel: UILabel!
+    @IBOutlet weak var searchTags: UISearchBar!
+    @IBOutlet weak var collectionView: UICollectionView!
+    @IBOutlet weak var addButton: FABView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        viewModel = ManageTagsViewModel()
+        
+        initUI()
+        initCollectionView()
+        viewModel?.viewDidLoad()
+        
+        NotificationCenter.default.addObserver(forName: ManageTagsViewModel.didUpdateTagsNotification, object: nil, queue: nil) { [weak self] notif in
+            self?.updateTagsNumber()
+            self?.collectionView.reloadData()
+        }
+    }
+    
+    func initUI() {
+        title = "Manage Tags".localized()
+        
+        archiveTitleNameLabel.font = Text.style35.font
+        archiveTitleNameLabel.textColor = .darkBlue
+        
+        archiveTitleTagsNbrLabel.font = Text.style34.font
+        archiveTitleTagsNbrLabel.textColor = .lightGray
+        
+        if let archiveName = AuthenticationManager.shared.session?.selectedArchive?.fullName {
+            archiveTitleNameLabel.text = "The \(archiveName) Archive Tags"
+            updateTagsNumber()
+        } else {
+            archiveTitleNameLabel.text = "Loading..."
+        }
+        
+        searchTags.updateHeight(height: 40, radius: 2)
+        if #available(iOS 13.0, *) {
+            searchTags.searchTextField.font = Text.style39.font
+        }
+        searchTags.setPlaceholderTextColor(.lightGray)
+        searchTags.tintColor = .lightGray
+        searchTags.searchTextPositionAdjustment = UIOffset(horizontal: 8, vertical: 0)
+        searchTags.barTintColor = .white
+        searchTags.backgroundColor = .white
+        
+        addButton.delegate = self
+        searchTags.delegate = self
+        
+        addButton.isHidden = true
+    }
+    
+    func initCollectionView() {
+        collectionView.register(ArchiveSettingsTagCollectionViewCell.nib(), forCellWithReuseIdentifier: ArchiveSettingsTagCollectionViewCell.identifier)
+        collectionView.register(ArchiveSettingsTagsHeaderCollectionView.nib(), forSupplementaryViewOfKind: ArchiveSettingsTagsHeaderCollectionView.kind, withReuseIdentifier: ArchiveSettingsTagsHeaderCollectionView.identifier)
+        
+        collectionView.backgroundColor = .clear
+        collectionView.indicatorStyle = .white
+        
+        let flowLayout = UICollectionViewFlowLayout()
```

Comment:
I feel like this is not needed. Usually the flow layout is the default layout set by the storyboard.